spec/spec_helper.rb

```diff
@@ -1,8 +1,9 @@
-require 'capybara/poltergeist'
+require 'capybara'
 
-Capybara.javascript_driver = :poltergeist
+Capybara.javascript_driver = :selenium
 
 RSpec.configure do |config|
+  Dir["./spec/support/**/*.rb"].sort.each { |f| require f }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.